denizen_scripts/relay/repo-link/Tasks/Repository_DCommand.dsc

```diff
@@ -0,0 +1,17 @@
+Repository_DCommand:
+    type: task
+    PermissionRoles:
+    - Everyone
+    definitions: Message|Channel|Author|Group
+    debug: false
+    Context: Color
+    script:
+# - ██ [ Clean Definitions & Inject Dependencies ] ██
+    # - inject Role_Verification
```

Comment:
Role verification isn't needed when it's marked for `everyone` for both discord groups.
Additional to the previous comment, we should only pass the `channel` definition until/for-if future additions are added.
```suggestion
    definitions: Channel
    debug: false
    Context: Color
    script:
# - ██ [ Clean Definitions & Inject Dependencies ] ██
```